src/main/java/seedu/address/logic/Messages.java

```diff
@@ -24,6 +24,7 @@ public class Messages {
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_HAS_CLASHES = "\nYou have %d other students with clashing schedule:\n%s";
     public static final String MESSAGE_REMINDER = "Reminder(s) for %s:\n";
+    public static final String MESSAGE_LIMIT = "The input/output exceeds the max allowable limit of 9,999,999.99";
```

Comment:
Can specify that this is the maximum value of PaidAmount that can be stored
![image](https://github.com/user-attachments/assets/6357e1e1-747a-433d-9748-d247d6c67f75)
This is what I did, what do you think about it? I think we need to update the error message for `AddCommand` when the input `Rate` is invalid also

Reply:
Yes, would be better to refer to the defined value

Reply:
Thanks for pointing this out! Ive made the changes